integration_tests/pom.xml

```diff
@@ -78,6 +78,42 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>clean-previously-populated-config-file</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>rm</executable>
+                            <arguments>
+                                <argument>-f</argument>
+                                <argument>${spark.rapids.source.basedir}/${rapids.module}/target/spark-rapids-default-configs.json</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
```

Comment:
since we write under target. we should defer to the `clean` phase to remove the file

```suggestion
```

Reply:
👍 Makes sense. Removed this.